SDK/AppCenterPush/Microsoft.AppCenter.Push.UWP/Push.cs

```diff
@@ -21,6 +22,10 @@ namespace Microsoft.AppCenter.Push
     public partial class Push
     {
         private PushNotificationChannel _channel;
+        private static readonly object UWPPushLock = new object();
```

Comment:
Changing in UWP means, it;s not available in other application type e.g. WPF?  should it be in Microsoft.AppCenter.Push.Shared? check once with @guperrot.

Reply:
Push is supported only in UWP.

Reply:
Can this just be called Lock?